consensus/scp/src/node.rs

```diff
@@ -138,12 +133,8 @@ pub trait ScpNode<V: Value>: Send {
     /// Get local node quorum set.
     fn quorum_set(&self) -> QuorumSet;
 
-    /// Submit a list of values of nomination.
-    fn nominate(
-        &mut self,
-        slot_index: SlotIndex,
```

Comment:
The `slot_index` argument was removed, because it only makes sense to nominate values for the current slot.